config/locales/de/models.yml

```diff
@@ -223,19 +223,19 @@ de:
           meetup: "Meetup"
           eventbrite: "Eventbrite"
           facebook: "Facebook"
-        registration_notification: "Registration Notifications" # TODO: Translate
-        registration_notifications: # TODO: Translate
-          digest: "One Email per Day"
-          immediate: "Immediately"
-          disabled: "No Notifications"
+        registration_notification: "Anmeldung Benachrichtigungen"
+        registration_notifications:
+          digest: "Einmal pro Tag"
+          immediate: "sofort"
```

Comment:
This is a title, it should be capitalised.

Reply:
will be Sofort